### ISSUE TITLE
Update javadoc boilerplate for experimental classes which are for public use

### ISSUE THREAD

**Issue by jack-berg** [open-telemetry/opentelemetry-java#6662]
We have standard boiler plate for internal classes which are public which reads:
```
 * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
 * at any time.
```
But I wonder if we should adjust our boilerplate for these types of methods - internal because we make no compatibility guarantees, but acceptable for public consumption provided the user accepts the risks of breaking changes.

Maybe something like:
```
This class is internal and experimental. Its APIs are unstable and can change at any time. Its APIs (or a version of them) may be promoted to the public stable API in the future, but no guarantees are made.
```

Places this updated message would be relevant:
- SdkTracerProviderUtil
- SdkMeterProviderUtil
- MeterConfig, TracerConfig, LoggerConfig
- SdkEventLoggerProvider
- ... and more

_Originally posted by @jack-berg in https://github.com/open-telemetry/opentelemetry-java/pull/6367#discussion_r1723676368_

**Comment by ravi-nagendran** [comment]
Hello, I am working on this.

**Comment by jack-berg** [comment]
Resolved in #6886.
